tests/widget/modal/widget-modal.login-social-redirect.test.js

```diff
@@ -6,7 +6,7 @@ test('Modal widget with social callback redirect', async ({ page }) => {
   const loginButton = page.getByRole('button', { name: 'Open Login Modal' });
   await loginButton.click();
 
-  const button = await page.locator('button', { hasText: 'Sign in with Google' });
+  const button = await page.locator('button', { hasText: 'Continue with Google' });
```

Comment:
I just updated the test here for the new default text for the social IDP button - whilst I was in the area though I had a think about e2e and whether we need additional use cases for this story. I am leaning towards there being enough coverage as it is - but please let me know if you agree / disagree with this, and what you feel any lack of coverage would be.

Reply:
I'm good with this being the only one.